src/plugins/query_enhancements/public/datasets/s3_type.test.ts

```diff
@@ -154,33 +152,6 @@ describe('s3TypeConfig', () => {
       expect(result.children?.[0].title).toBe('DataSource 1');
       expect(result.hasNext).toBe(true);
     });
-
-    it('should filter out data sources with versions lower than 1.0.0', async () => {
```

Comment:
nit: perhaps its worth to keep this test to but flip it so it shows that we do NOT filter out data sources